### PR TITLE
Fix several TODOs

### PR DESCRIPTION
**Pull request by martmists**

### CODE REVIEW

File: src/main/kotlin/me/alexflipnote/kawaiibot/commands/Slap.kt [hunk 1]
```diff
@@ -19,7 +20,7 @@ class Slap : ICommand {
             m.user.idLong == ctx.jda.selfUser.idLong ->
                 ctx.send("**${ctx.author.name}** we can no longer be friends ;-;")
             m.user.idLong == ctx.author.idLong ->
-                ctx.send("https://media0.giphy.com/media/1M9fmo1WAFVK0/giphy.gif") // TODO: bundle this maybe
+                ctx.channel.sendFile(Helpers.getImageStream("images/butwhy.webp"), "butwhy.webp").queue()
             else -> {
```
Comment (AlexFlipnote) [inline src/main/kotlin/me/alexflipnote/kawaiibot/commands/Slap.kt]: huh.. webp?
I know what it is but, I don't think Apple users (specially iPhone/iPad) supports this format

**Comment by martmists** [comment]
Will convert it to gif :thumbsup:

**Comment by martmists** [comment]
I'll fix this fully when I get to linux

File: src/main/kotlin/me/alexflipnote/kawaiibot/commands/Ban.kt [hunk 2]
```diff
@@ -17,7 +17,7 @@ class Ban : ICommand {
         }
         val reason = ctx.args.collect()
         ctx.guild.controller
-                .ban(user.toString(), 0) // TODO: wat, doesnt accept Longs, jda bug?
+                .ban(member, 0)
```
Comment (aurieh) [inline src/main/kotlin/me/alexflipnote/kawaiibot/commands/Ban.kt]: This does not work on unresolved members. Please revert this change

File: src/main/kotlin/me/alexflipnote/kawaiibot/commands/Lick.kt [hunk 3]
```diff
@@ -20,8 +21,7 @@ class Lick : ICommand {
             m.user.idLong == ctx.jda.selfUser.idLong ->
                 ctx.send("${ctx.author.name}... w-why do you lick me ;-;")
             m.user.idLong == ctx.author.idLong ->
-                // TODO: download and bundle this image
-                ctx.send("https://blog.eat24.com/wp-content/uploads/2016/03/cat-licking-itself-.gif")
+                ctx.channel.sendFile(Helpers.getImageStream("images/selflick.gif"), "selflick.gif").queue()
```
Comment (Ratismal) [inline src/main/kotlin/me/alexflipnote/kawaiibot/commands/Lick.kt]: Why not use an embed the original URL? It would be less resource intensive than uploading the image every time.

File: src/main/kotlin/me/alexflipnote/kawaiibot/commands/Slap.kt [hunk 4]
```diff
@@ -19,7 +20,7 @@ class Slap : ICommand {
             m.user.idLong == ctx.jda.selfUser.idLong ->
                 ctx.send("**${ctx.author.name}** we can no longer be friends ;-;")
             m.user.idLong == ctx.author.idLong ->
-                ctx.send("https://media0.giphy.com/media/1M9fmo1WAFVK0/giphy.gif") // TODO: bundle this maybe
+                ctx.channel.sendFile(Helpers.getImageStream("images/butwhy.webp"), "butwhy.webp").queue()
```
Comment (Ratismal) [inline src/main/kotlin/me/alexflipnote/kawaiibot/commands/Slap.kt]: Again, why not embed?

File: src/main/kotlin/me/alexflipnote/kawaiibot/commands/Lick.kt (hunk 3 above)
Comment (martmists) [reply src/main/kotlin/me/alexflipnote/kawaiibot/commands/Lick.kt]: It's better to be consistent with the other commands which also upload their own images every time in my opinion, changing all those would be a completely different PR.

Comment (Ratismal) [reply src/main/kotlin/me/alexflipnote/kawaiibot/commands/Lick.kt]: 👌

Comment (Devoxin) [reply src/main/kotlin/me/alexflipnote/kawaiibot/commands/Lick.kt]: ^Plus links look somewhat unclean. Imagine responses like `Oh you're all alone? Sorry to hear <link to image of gif>`

Comment (Ratismal) [reply src/main/kotlin/me/alexflipnote/kawaiibot/commands/Lick.kt]: I think you misunderstood what I meant by "embed". I don't mean just plopping the URL down, I mean using the actual rich embed structure that discord provides.
